libraries/apollo-gradle-plugin-external/src/main/kotlin/com/apollographql/apollo3/gradle/internal/DefaultApolloExtension.kt

```diff
@@ -57,7 +56,7 @@ abstract class DefaultApolloExtension(
   internal fun getServiceInfos(project: Project): List<ApolloGradleToolingModel.ServiceInfo> = services.map { service ->
     DefaultServiceInfo(
         name = service.name,
-        schemaFiles = service.lazySchemaFiles(project),
+        schemaFiles = service.schemaFilesSnapshot(project),
```

Comment:
I never realized that but there is potential for failure here. Is `getServiceInfos()` depending on other tasks? If `schemaFiles` is generated by another task vs being always commited in the repo, the IJ plugin might see an absent file.

Reply:
This is called directly inside `ApolloPlugin.apply`, after the creation of `ApolloExtension` - so not depending on any tasks. Not sure what we could do to make this scenario work though 🤔